src/aiori-POSIX.c

```diff
@@ -427,6 +428,24 @@ void *POSIX_Create(char *testFileName, IOR_param_t * param)
         return ((void *)fd);
 }
 
+/*
+ * Creat a file through mknod interface.
+ */
+void *POSIX_Mknod(char *testFileName)
+{
+	int *fd;
+
+	fd = (int *)malloc(sizeof(int));
```

Comment:
This is not optimal, as with Mknod you do not call POSIX_close() which calls  free(fd). I will fix this, though in a subsequent patch.